jsonnet/jsonnet.bzl

```diff
@@ -107,8 +114,8 @@ def _jsonnet_to_json_impl(ctx):
       ["-J .",
        "-J %s" % ctx.genfiles_dir.path,
        "-J %s" % ctx.bin_dir.path] +
-      ["--ext-str '%s'='%s'"
-       % (var, ctx.expand_make_variables("vars", jsonnet_ext_strs[var],{})) for var in jsonnet_ext_strs.keys()] +
+      ["--ext-str %s=%s"
+       % (_quote(key), _quote(ctx.var[val])) for key, val in jsonnet_ext_strs.items()] +
```

Comment:
This breaks the expected behavior according to the docs. The docs make it seem like you can pass a string literal, but this expects a variable to exist. If not, it crashes.

Perhaps we can settle on using ext_str_env for variables and go back to string literals here without the deprecated make variables?